lib/api/jenkinsInfo.js

```diff
@@ -0,0 +1,14 @@
+var auth = require("../util/auth")
+
+// Retrieve jenkins server information
```

Comment:
Just an observation: the other doc is written with block format (/** ... */) but here is line format (// ...)